source/content/guides/multisite/01-introduction.md

```diff
@@ -20,10 +20,10 @@ editpath: multisite/01-introduction.md
 image: multisite
 guidetoc: true
 ---
-This guide covers the process of requesting, configuring, and maintaining a new WordPress Site Network on Pantheon. We'll also explore supported use cases, considerations and some common troubleshooting tips.
+This guide covers the process of requesting, configuring, and maintaining a new WordPress Multisite on Pantheon. We'll also explore supported use cases, considerations and some common troubleshooting tips.
 
-## About WordPress Site Networks
-Pantheon supports [WordPress Site Networks](https://codex.wordpress.org/Glossary#Network) (also known as WordPress Multisite) which lets you create a network of sites using a single copy of the WordPress codebase and a common database. For those responsible for maintaining several or dozens of similar sites, WordPress Site Networks can make it much easier to fix bugs and deploy new features across all of those sites.
+## About WordPress Multisite
+Pantheon supports [WordPress Multisite](https://wordpress.org/documentation/article/wordpress-glossary/#multisite) (also known as WordPress Site Network) which lets you create a network of sites using a single copy of the WordPress codebase and a common database. For those responsible for maintaining several or dozens of similar sites, WordPress Multisite can make it much easier to fix bugs and deploy new features across all of those sites.
```

Comment:
Nope, hard pass on walking back our decision to rename everything "multisite" over "site network". That language at all is confusing for WordPress users and Pantheors alike. A definition for multisite is already linked, so there's no need to even have a reference to "site network" except for people who understand the history of "WordPress Site Network" terminology at Pantheon (and no one besides us is paying _that_ much attention).

```suggestion
Pantheon supports [WordPress Multisite](https://wordpress.org/documentation/article/wordpress-glossary/#multisite) which lets you create a network of sites using a single copy of the WordPress codebase and a common database. For those responsible for maintaining several or dozens of similar sites, WordPress Multisite can make it much easier to fix bugs and deploy new features across all of those sites.
```